### PR TITLE
Turns out I had in fact messed up some of the CSS

### PR DESCRIPTION
**Pull request by dambem**
## Description of this PR
So.. It turns out I did also mess up some of the CSS by constantly spamming updates (sorry @M-Davies!) 

However, this new PR should fix that, and also made the sidebar slightly smaller (just so more of the content is on screen)

## Checklist
- [x ] Tests all pass

### CODE REVIEW

**Comment by AlvaroRausell** [comment]
@dambem FYI, you committed the map API key

File: ClimateMonitorV2/public/javascripts/config.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // PLEASE ASK DAMIAN FOR ANY KEYS YOU DON'T HAVE
 var config = {
-    MAP_KEY: "",
-    WEATHER_COMPANY_KEY: "",
-    GITHUB_API_TOKEN: ""
+    MAP_KEY: "pk.eyJ1IjoiZGFtYmVtIiwiYSI6ImNrYjJnbjQ1djBvbTkzMmxvMTFpZ2lvMWEifQ.jwAs_GyT8Q1Rhu8NAPcJYA",
```
Comment (AlvaroRausell) [inline ClimateMonitorV2/public/javascripts/config.js]: Remove

**Review by AlvaroRausell — DISMISSED** [review]

**Comment by Megan-Wright** [comment]
@dambem ran this just now and LGTM other than the map key and merge conflicts :)

**Comment by dambem** [comment]
Argh, accidentally deleted it like a damn fool,

**Review by M-Davies — APPROVED** [review]
Looks good (except for the conflicts). Thanks for fixing this :)

**Pull request merged**